hipstersmoothie/storybook
# Update Node.js

Issue: #4224

## What I did

Change the font color to be accessible

## How to test

Is this testable with Jest or Chromatic screenshots? Yes
Does this need a new example in the kitchen sink apps? No
Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
